packages/@o3r/core/schematics/service/index.ts

```diff
@@ -54,26 +54,45 @@ export function ngGenerateService(options: NgGenerateServiceSchematicsSchema): R
       packageName = JSON.parse(tree.read(path.resolve(destination, rec, 'package.json'))!.toString()).name?.split('/')[0] || destination;
     }
 
-    const templateSource = apply(url('./templates'), [
-      template({
-        ...strings,
-        ...options,
-        featureName,
-        currentServiceIndex,
-        currentFixtureJasmineIndex,
-        currentFixtureJestIndex,
-        serviceName: strings.capitalize(strings.camelize(options.name + ' ' + options.featureName)),
-        packageName
-      }),
+    const templateData = {
+      ...strings,
+      ...options,
+      featureName,
+      currentServiceIndex,
+      currentFixtureJasmineIndex,
+      currentFixtureJestIndex,
+      serviceName: strings.capitalize(strings.camelize(options.name + ' ' + options.featureName)),
+      packageName
+    };
+
+    const baseTemplateSource = apply(url('./templates/base'), [
+      template(templateData),
       renameTemplateFiles(),
       move(destination)
     ]);
+
+    const rules = [mergeWith(baseTemplateSource, MergeStrategy.Overwrite)];
+
+    ['jest', 'jasmine-core'].forEach(testFramework => {
+      try {
+        require.resolve(`${testFramework}/package.json`);
```

Comment:
Could be good to add test case 
even if it's not testable for now because of [this jest issue](https://github.com/jestjs/jest/issues/9543)